compiler/src/iree/compiler/Dialect/Flow/Transforms/AnnotateDispatches.cpp

```diff
@@ -80,7 +83,11 @@ static int64_t estimateLinalgExtOpCost(Operation *op) {
   // This is something like the extra log(N) factor for a sort or FFT, or
   // the amount of work done by a softmax vs a cheap elementwise on a tensor
   // of the same shape.
-  cost *= 10;
+  if (cost >= kMaxCost / 10) {
+    cost = kMaxCost;
+  } else {
+    cost *= 10;
+  }
```

Comment:
I thought that there is a std function can be used, then I found that it is c++26. :p (what, c++26!?)

https://en.cppreference.com/w/cpp/numeric/mul_sat